intents-operator/templates/intents-operator-allow-access-to-webhook-netpol.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
can we split this and add it to the relevant deployed object? 
i.e. put the netpol for port 2112 along with intents-operator-controller-manager-metrics-service.yaml, etc.